app/models/neighbourhood.rb

```diff
@@ -1,6 +1,10 @@
 # frozen_string_literal: true
 
 class Neighbourhood < ApplicationRecord
+  # WARNING: this must be updated for every new ONS dataset
```

Comment:
this should probably be in the documentation md file

Reply:
[like here](https://github.com/geeksforsocialchange/PlaceCal/blob/ik-2017-duplicate-neighbourhoods/doc/updating-neighbourhoods.md#adjust-the-neighbourhood-model)?